lms/services/ltia_http.py

```diff
@@ -58,10 +63,13 @@ def _get_access_token(self, scopes):
 
 
 def factory(_context, request):
+    lti_registration = (
+        request.find_service(name="application_instance").get_current().lti_registration
+    )
+
     return LTIAHTTPService(
-        request.find_service(name="application_instance")
-        .get_current()
-        .lti_registration,
+        lti_registration,
+        request.product.plugin.misc.get_ltia_aud_claim(lti_registration),
```

Comment:
Injecting the value directly rather than using the plugin in the service.

Reply:
You are right, it doesn't matter right now but the usual pattern it's to inject the service.

I was a bit hesitant to depend on the "Misc" plugin but if this becomes "LTIAPlugin" it just a change on the interface.